tests/integration/test_logcollector/test_statistics/test_statistics_macos.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+import pytest
+import wazuh_testing.tools.configuration as conf
+
+from json import load
+from wazuh_testing import logcollector
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools import LOGCOLLECTOR_STATISTICS_FILE
+from wazuh_testing.tools.services import control_service
+from wazuh_testing.tools.file import read_json
+
+# Marks
+pytestmark = [pytest.mark.darwin, pytest.mark.tier(level=1)]
+
+# Configuration
+logcollector_stats_file_tout = 10
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_statistics_macos.yaml')
+
+parameters = [
+    {'LOCATION': 'macos', 'LOG_FORMAT': 'macos'},
+]
+
+metadata = [
+    {'location': 'macos', 'log_format': 'macos'}
+]
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f"{x['LOCATION']}_{x['LOG_FORMAT']}" for x in parameters]
+
+local_internal_options = {'logcollector.state_interval': 1}
+
+daemons_handler_configuration = {'daemons': ['wazuh-logcollector'], 'ignore_errors': False}

```

Comment:
```suggestion
daemons_handler_configuration = {'daemons': ['wazuh-logcollector', 'wazuh-agentd', 'wazuh-execd'], 'ignore_errors': False}

```

Reply:
After testing, we saw that these daemons are needed in order for the test to pass. It was passing before due to the fixture `configure_local_internal_options_logcollector ` set in `test_logcollector/conftest.py` that was leaving the daemons started.